boefjes/plugins/kat_log4shell/main.py

```diff
@@ -15,30 +15,29 @@
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 logging.basicConfig(level=logging.INFO)
 
-TIMEOUT = 15
-REPLY_FQDN = getenv("REPLY_FQDN", "invalid")  # "cve.stillekat.nl"
-
 
 def run(boefje_meta: BoefjeMeta) -> Tuple[BoefjeMeta, Union[bytes, str]]:
     input_ = boefje_meta.arguments["input"]
     host = input_["name"]
-
     identifier = boefje_meta.id
-
     schemes = ["http", "https"]
+    timeout = 15
 
-    reply_fqdn = REPLY_FQDN.lower()
+    reply_fqdn_env = getenv("REPLY_FQDN", "invalid")
+    reply_fqdn = reply_fqdn_env.lower()
```

Comment:
Not if you want the original value of `REPLY_FQDN` in the `ValueError` (otherwise it might create confusion for the person who decided to use captials in a FQDN and sees something different in the error message)